SimPEG/dask/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -1,22 +1,23 @@
+import dask.array as da
+
 from .....electromagnetics.static.induced_polarization.simulation import (
     BaseIPSimulation as Sim,
 )
 
-import dask.array as da
-
 
 def dask_getJtJdiag(self, m, W=None, f=None):
     """
     Return the diagonal of JtJ
     """
     if getattr(self, "_gtgdiag", None) is None:
+        J = self.getJ(m, f=f)
```

Comment:
```suggestion
```

Reply:
keep J as there is no Jmatrix I believe in vanilla SimPEG